cnd/src/swap_protocols/halight/connector.rs

```diff
@@ -158,13 +158,14 @@ impl LndConnectorAsSender {
         &self,
         secret_hash: SecretHash,
         status: PaymentStatus,
-    ) -> Result<Option<Payment>, Error> {
+    ) -> anyhow::Result<Option<Payment>> {
         let response = client(&self.certificate, &self.macaroon)?
             .get(self.payment_url())
             .send()
             .await?
```

Comment:
Do we want to add context here as-well?

Reply:
My assumption here was that the payment URL will be part of the error returned by lnd. Do you think that's incorrect or would you add something else?

Reply:
I don't think it is going to be part of it :)